proto/archway/rewards/v1beta1/tx.proto

```diff
@@ -26,3 +31,17 @@ message MsgSetContractMetadata {
 
 // MsgSetContractMetadataResponse is the response for Msg.SetContractMetadata.
 message MsgSetContractMetadataResponse {}
+
+// MsgWithdrawRewards is the request for Msg.WithdrawRewards.
+message MsgWithdrawRewards {
+  // rewards_address is the address to distribute rewards to (bech32 encoded).
+  string rewards_address = 2;
+}
+
+// MsgWithdrawRewardsResponse is the response for Msg.WithdrawRewards.
+message MsgWithdrawRewardsResponse {
+  // rewards are the total rewards transferred.
+  repeated cosmos.base.v1beta1.Coin rewards = 3 [
```

Comment:
We should return `RewardsRecord` here as height and time might be useful for distribution contracts.

Reply:
So we return all the records we have without aggregation (sum of all rewards), right?